CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v1.3.0-0.22b0...HEAD)
 
+### Added
+- Remove `opentelemetry-instrumentation`
```

Comment:
This should make clear that the package was moved to the core repo

Reply:
Fixed